lib/modules/apostrophe-palette-admin-bar/index.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  improve: 'apostrophe-admin-bar',
+  addNotSafeLiveList: ['apostrophe-palette']
```

Comment:
this will get blown out if the developer has their own instance of this option. Better to use the beforeConstruct concat pattern.